.github/workflows/link_and_spell_check.yml

```
@@ -50,3 +50,8 @@ jobs:
         strict: false
         incremental_files_only: true
         config: 'cspell.json'
+
+  check-type-annotations:
+  - name: "Check type annotations and mypy stub files"
+    run: ./build.ps1 -t checkmypy
+    shell: pwsh
```

The check isn't running, I think we need to move this to its own job with steps
```suggestion
  check-type-annotations:
    runs-on: ubuntu-latest
    name: "Check type annotations and mypy stub files"
    steps:
      - uses: actions/checkout@v3
      - name: "Run mypy"
        run: ./build.ps1 -t checkmypy
        shell: pwsh

```